### PR TITLE
[Feature]:-Added the blog page and blogs in it

### LINKED ISSUE #12: [Frontend] : Implement `/blog` route and component on the website
Just like standard website, We want to have a blog section where we can spin the articles related to nurse industry. While implementing the blog page you can keep lorem epsum dummy text and make sure it's somewhere in JSON. This will help us in future to add blogs directly from the backend infrastructure.

The design is upto the contributor. We have no objections and would love to see great ideas incoming.

### PR DESCRIPTION
**Pull request by mohit-bhandari45**
Fixes #12

Description:-
So the requirement was to add the blog page where all the blogs will be rendered through the backend. So did I do.
All the blogs will rendered from the database in the form of json. All  the blogs are stored in the react state.
And by react mapping functions all the blogs are shown in the ui.
They are in this form:-
{
            "title": "Lorem Ipsum Blog",
            "content": "Lorem ipsum dolor sit amet, consectetur adipiscing elit...",
            "author": "John Doe",
            "publication_date": "2024-05-03"
},

Screenshots:-
image
image

### CODE REVIEW

**Comment by Harshil-Jani** [comment]
@mohit-bhandari45 Great Job. Thanks

Would you like to add another commit where you can just put it in header saying "blog" besides "career" and also add the route link in the footer where it redirects to "/blog" ?

Also please avoid commiting package-lock.json and yarn.lock since the PR doesn't add or remove any dependencies.

File: package-lock.json
Comment (Harshil-Jani) [inline package-lock.json:1]: Don't push `package-lock.json`

File: yarn.lock
Comment (Harshil-Jani) [inline yarn.lock:1]: Don't push `yarn.lock`

**Review by Harshil-Jani — CHANGES_REQUESTED** [review]

**Comment by mohit-bhandari45** [comment]
done you can check now

**Comment by mohit-bhandari45** [comment]
changes done you can check it out

**Review by Harshil-Jani — APPROVED** [review]
Thanks @mohit-bhandari45

**Pull request merged**